.travis.yml

```diff
@@ -64,8 +64,8 @@ script:
   - flake8
   - black --check --diff .
   - python manage.py check_for_endpoint_documentation
-  - pytest --ignore-glob='**/tests/integration/*' --cov=usaspending_api --cov-report= --reuse-db -rsx
-  - pytest --override-ini=python_files='**/tests/integration/test_*.py **/tests/integration/*_test.py' --cov=usaspending_api --cov-append --cov-report term --cov-report xml:coverage.xml --reuse-db -rsx
+  - pytest --durations 50 --ignore-glob='**/tests/integration/*' --cov=usaspending_api --cov-report= --reuse-db -rsx
```

Comment:
Are these changes intentionally in the diff?

Reply:
Yes.  They will always show the 50 slowest tests so we can shame the original developer into speeding them up.